packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -59,19 +64,16 @@ export class QuoteRequestor {
                         params: {
                             sellToken,
                             buyToken,
-                            buyAmount: marketOperation === MarketOperation.Buy ? assetFillAmount.toString() : undefined,
-                            sellAmount:
-                                marketOperation === MarketOperation.Sell ? assetFillAmount.toString() : undefined,
+                            sellAmount: assetFillAmount.toString(),
                             takerAddress,
                         },
                         timeout: makerEndpointMaxResponseTimeMs,
                     });
                 } catch (err) {
                     logUtils.warn(
-                        `Failed to get RFQ-T quote from market maker endpoint ${rfqtMakerEndpoint} for API key ${takerApiKey} for taker address ${takerAddress}: ${JSON.stringify(
-                            err,
-                        )}`,
+                        `Failed to get RFQ-T quote from market maker endpoint ${rfqtMakerEndpoint} for API key ${takerApiKey} for taker address ${takerAddress}`,
                     );
+                    logUtils.warn(err);
```

Comment:
`JSON.stringify(err)` which was used in the code outputted `{}` for an actual error.  I needed to add this line to actually see any error thrown